custom_dc/faq.md

```diff
@@ -19,9 +19,12 @@ Yes; there are many options for doing so. If you want an entirely private site w
 
 Note that you cannot apply fine-grained access restrictions, such as access to specific data or pages. Access is either all or nothing. If you want to be able to partition off data, you would need to create additional custom instances.
 
-### Will my custom data end up in the base Data Commons?
+### Will my data or queries end up in base Data Commons? {#data-security}
 
-No. Even if a query needs data from the base Data Commons, gleaned from the (incomplete) results from your custom data, it will never transfer any custom data to the base data store.
+Your user queries, observations data, or property values are never transferred to base Data Commons. The NL model built from your custom data lives solely in your custom instance. The custom Data Commons Mixer does make API calls to the base Data Commons Mixer (as depicted in [this diagram](/custom_dc/index.html#system-overview)) only in the following instances:
+- At data load time, an API call is made from the custom instance to the base instance to resolve entity names to DCIDs; for example, if your data refers to a particular country name, the custom instance will send an API request to look up its DCID. 
```

Comment:
Again, have we introduced DCID or should we have a blurb about mapping entities to their respective identifiers in DC?

Reply:
Yes, it's mentioned several times in https://docs.datacommons.org/custom_dc/custom_data.html. But I'll add a link here to the glossary entry.